src/test/java/uk/ac/ebi/ena/webin/cli/WebinCliTest.java

```diff
@@ -531,14 +531,15 @@ public void testPrintManifestHelpReads() {
                 "│SUBMISSION_TOOL_VERS│Optional   │Version number of the third-party or         │\n" +
                 "│ION                 │           │developed tool used to submit to ENA         │\n" +
                 "├────────────────────┼───────────┼─────────────────────────────────────────────┤\n" +
-                "│FASTQ               │0-2 files  │Fastq file                                   │\n" +
+                "│FASTQ               │0-999 files│Fastq file                                   │\n" +
```

Comment:
Thought about this more this morning and I suggest that we cap the number Fastq files to 10 to start with.